nextflow.config

```
@@ -1,47 +1,198 @@
+    arg_amrfinderplus_db                    = null
+    arg_amrfinderplus_identmin              = -1
+    arg_amrfinderplus_coveragemin           = 0.5
+    arg_amrfinderplus_translationtable      = 11
+    arg_amrfinderplus_plus                  = false
+    arg_amrfinderplus_name                  = false
+
+    arg_skip_deeparg                        = false
+    arg_deeparg_data                        = null
+    arg_deeparg_data_version                = 2     // Make sure to update on module version bump!
+    arg_deeparg_model                       = 'LS'
+    arg_deeparg_minprob                     = 0.8
+    arg_deeparg_alignmentidentity           = 50
+    arg_deeparg_alignmentevalue             = 1E-10
+    arg_deeparg_alignmentoverlap            = 0.8
+    arg_deeparg_numalignmentsperentry       = 1000
+
+    arg_skip_abricate                       = false
+    arg_abricate_db                         = 'ncbi'
+    arg_abricate_minid                      = 80
+    arg_abricate_mincov                     = 80
+
+    // BGC options
+    run_bgc_screening                       = false
+
+    bgc_skip_antismash                      = false
+    bgc_antismash_databases                 = null
+    bgc_antismash_installationdirectory     = null
+    bgc_antismash_cbgeneral                 = false
+    bgc_antismash_cbknownclusters           = false
+    bgc_antismash_cbsubclusters             = false
+    bgc_antismash_smcogtrees                = false
+    bgc_antismash_ccmibig                   = false
+    bgc_antismash_contigminlength           = 1000
+    bgc_antismash_hmmdetectionstrictness    = 'relaxed'
+    bgc_antismash_taxon                     = 'bacteria'
+    bgc_antismash_sampleminlength           = 1000
+
+    bgc_skip_deepbgc                        = false
+    bgc_deepbgc_database                    = null
+    bgc_deepbgc_score                       = 0.5
+    bgc_deepbgc_prodigalsinglemode          = false
+    bgc_deepbgc_mergemaxproteingap          = 0
+    bgc_deepbgc_mergemaxnuclgap             = 0
+    bgc_deepbgc_minnucl                     = 1
+    bgc_deepbgc_minproteins                 = 1
+    bgc_deepbgc_mindomains                  = 1
+    bgc_deepbgc_minbiodomains               = 0
+    bgc_deepbgc_classifierscore             = 0.5
+
+    bgc_skip_gecco                          = false
+    bgc_gecco_cds                           = 3
+    bgc_gecco_threshold                     = 0.8
+    bgc_gecco_pfilter                       = 1E-9
+    bgc_gecco_edgedistance                  = 0
+    bgc_gecco_mask                          = false
+
+    bgc_skip_hmmsearch                      = false
+    bgc_hmmsearch_models                    = null
+    bgc_hmmsearch_savealignments            = false
+    bgc_hmmsearch_savetargets               = false
+    bgc_hmmsearch_savedomains               = false
+
+    // Reporting options
+    params.arg_hamronization_summarizeformat = 'tsv'
```

This looks wrong :eyes: 

```suggestion
    arg_hamronization_summarizeformat = 'tsv'
```

Can we check we are actually using this in the pipeline, and also move it up to the ARG section (it looks like ampCOMBI options are also grouped in that section)